AsyncDisplayKit/ASTableView.h

```diff
@@ -272,20 +272,30 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)moveRowAtIndexPath:(NSIndexPath *)indexPath toIndexPath:(NSIndexPath *)newIndexPath;
 
 /**
- * Similar to -cellForRowAtIndexPath:.
+ * Retrieves the node for the row at the given index path, in the data source's index space.
  * 
  * @param indexPath The index path of the requested node.
+ */
+- (nullable ASCellNode *)nodeForRowAtIndexPath:(NSIndexPath *)indexPath;
+
+/**
+ * Retrieves the node for the row at the given index path, in UITableView's index space.
  *
- * @return a node for display at this indexpath.
+ * @param indexPath The index path of the requested node.
  */
-- (ASCellNode *)nodeForRowAtIndexPath:(NSIndexPath *)indexPath;
+- (nullable ASCellNode *)nodeForRowAtCompletedIndexPath:(NSIndexPath *)indexPath;
```

Comment:
`nodeForRowAtAsynchronousIndexPath` ? Not any clearer without documentation but…